GPU/Vulkan/ShaderManagerVulkan.cpp

```diff
@@ -162,7 +162,7 @@ static void ConvertProjMatrixToVulkan(Matrix4x4 &in, bool invertedX, bool invert
 	if (invertedY)
 		yoff = -yoff;
 
-	const Vec3 trans(xoff, yoff, gstate_c.vpZOffset + 0.5f);
+	const Vec3 trans(xoff, yoff, gstate_c.vpZOffset * 0.5f + 0.5f);
```

Comment:
Wait wait wait, does Vulkan use the half-pixel offset insanity that Direct3D9 does?  It doesn't right?

-[Unknown]

-[Unknown]